content/20.pharmaceuticals.md

```diff
@@ -667,7 +675,8 @@ Reactivation of hepatitis B and herpes simplex virus 1 was also reported in a sm
 A July 2020 case report described negative outcomes of two COVID-19 patients after receiving TCZ, including one death; however, both patients were intubated and had entered septic shock prior to receiving TCZ [@doi:10.1016/j.chest.2020.04.024], likely indicating a severe level of cytokine production.
 Additionally, D-dimer and sIL2R levels were reported by one study to increase in patients treated with TCZ, which raised concerns because of the potential association between elevated D-dimer levels and thrombosis and between sIL2R and diseases where T-cell regulation is compromised [@doi:10.1016/j.chest.2020.06.006].
 An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported [@doi:10.2139/ssrn.3642653].
-TCZ administration to COVID-19 patients is not without risks, may introduce additional risk of developing secondary infections, and should be approached especially cautiously for patients who have latent viral infections.
+In the RECOVERY trial, however, only three out of 2,022 participants in the group receiving TCZ developed adverse reactions determined to be associated with the intervention, and no excess deaths were reported [@doi:10.1101/2021.02.11.21249258].
+TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; therefore, it should be approached especially cautiously for patients who have latent viral infections.
```

Comment:
It seems like the risks are not super high though, based on the available data?

Reply:
```suggestion
TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; however, while caution may be prudent when treating patients who have latent viral infections, the results of the RECOVERY trial indicate that adverse reactions to TCZ are very rare among COVID-19 patients broadly.
```